Piece.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include <iostream>
+#include <SFML/Graphics.hpp>
+#include <SFML/Network.hpp>
+//typy: 0pawn 1bishop 2knight 3rook 4queen 5king
+//kolory: 0bia�y 1czarny
+//nie uzywam enum�w �eby nie jeba� si� z overloadami << do pakiet�w
```

Comment:
Do biały/czarny może, ale nie musi być enum, ale typy są na to za duże.
Żeby to podzielić można zrobić klasę bazową "Piece" i od niej potem klasy dla figur.

Na pewno trzymanie takich informacji w komentarzu nie jest najlepszym pomysłem.